resource/generated_factories.cpp.em

```diff
@@ -281,4 +290,85 @@ Factory<
 }
 
 @[end for]@
+
+@[for service in services]@
+  @[for frm, to in [("1", "2"), ("2", "1")]]@
+    @[for type in ["Request", "Response"]]@
```

Comment:
Please use a similar indentation for these for loop as above in the file to ensure that the generated code is correctly indented.